app/celery/provider_tasks.py

```diff
@@ -69,6 +74,8 @@ def deliver_email(self, notification_id):
         current_app.logger.error(f"Cannot send notification {notification_id}, got an invalid direct file url.")
         update_notification_status_by_id(notification_id, NOTIFICATION_TECHNICAL_FAILURE)
         _check_and_queue_callback_task(notification)
+    except (MalwareDetectedException, DocumentDownloadException):
+        _check_and_queue_callback_task(notification)
```

Comment:
When is the `DocumentDownloadException` exception raised? It is new behavior, right? I wonder because before, we'd retry more times via `self.retry` before reporting a failure. Is this exception raised and cannot be recovered at all?